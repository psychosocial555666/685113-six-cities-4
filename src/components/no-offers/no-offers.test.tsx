import * as React from 'react';
import * as renderer from 'react-test-renderer';
import NoOffers from './no-offers';

it(`Should NoOffers render correctly`, () => {
  const tree = renderer.create(<NoOffers />).toJSON();

  expect(tree).toMatchSnapshot();
});
