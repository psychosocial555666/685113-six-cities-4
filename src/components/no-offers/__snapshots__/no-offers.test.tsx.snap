// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NoOffers render correctly 1`] = `
<div
  className="cities__places-container cities__places-container--empty container"
>
  <section
    className="cities__no-places"
  >
    <div
      className="cities__status-wrapper tabs__content"
    >
      <b
        className="cities__status"
      >
        No places to stay available
      </b>
      <p
        className="cities__status-description"
      >
        We could not find any property available at the moment in Dusseldorf
      </p>
    </div>
  </section>
  <div
    className="cities__right-section"
    style={
      Object {
        "backgroundImage": "url(/img/no-places@2x.png)",
        "backgroundPosition": "right center",
        "backgroundSize": "100% auto",
      }
    }
  />
</div>
`;
